backend/flow_api/flow/credential_usage/action_remember_me.go

```diff
@@ -0,0 +1,43 @@
+package credential_usage
+
+import (
+	"fmt"
+
+	"github.com/teamhanko/hanko/backend/flow_api/flow/shared"
+	"github.com/teamhanko/hanko/backend/flowpilot"
+)
+
+type RememberMe struct {
+	shared.Action
+}
+
+func (a RememberMe) GetName() flowpilot.ActionName {
+	return shared.ActionRememberMe
+}
+
+func (a RememberMe) GetDescription() string {
+	return "Show a remember me checkbox."
+}
+
+func (a RememberMe) Initialize(c flowpilot.InitializationContext) {
+	deps := a.GetDeps(c)
+
+	c.AddInputs(flowpilot.BooleanInput("remember_me").Required(true))
+
+	if deps.Cfg.Session.EnableRememberMe {
+		c.SuspendAction()
+	}
+}
+
+func (a RememberMe) Execute(c flowpilot.ExecutionContext) error {
+
+	if valid := c.ValidateInputData(); !valid {
+		return c.Error(flowpilot.ErrorFormDataInvalid)
+	}
+
+	if err := c.Stash().Set(shared.StashPathRememberMe, c.Input().Get(shared.StashPathRememberMe).Bool()); err != nil {
+		return fmt.Errorf("failed to set remember_me to stash: %w", err)
+	}
```

Comment:
Hi @lfleischmann, this action execute function seems to work fine but when I try to retrieve the value in issue session hook, the value seems to not exit in the stash

Reply:
Ok, let me try that

Reply:
Using this approach, create new csrf_token which leads csrf token mismatch error @lfleischmann

Reply:
It works